recipe/meta.yaml

```diff
@@ -42,8 +48,8 @@ about:
     effectiveness of tests. It can show which parts of your code are being
     exercised by tests, and which are not.
   doc_url: http://coverage.readthedocs.org
-  dev_url: https://bitbucket.org/ned/coveragepy
-  doc_source_url: https://bitbucket.org/ned/coveragepy/src/default/doc/?at=default
+  dev_url: https://github.com/nedbat/coveragepy
+  doc_source_url: https://github.com/nedbat/coveragepy/blob/master/doc/index.rst
```

Comment:
To properly reflect the state of the documentation, `doc_source_url` should be anchored to the release version and not to the tip of the `master` or `main` branch. Additionally, this URL should point to a directory (AKA git tree), rather than a specific file (since we're trying to provide the source URL for _all_ the documentation, rather than the documentation landing page).

Reply:
The `doc_source_url` was modified as requested: 

https://github.com/AnacondaRecipes/coverage-feedstock/blob/update_to_6.1.1/recipe/meta.yaml#L50&L52

However the example provided in `docs.conda.io` is pointing to an individual file, and also pointing to the master branch 

`doc_source_url:` https://github.com/shibukawa/imagesize_py/blob/master/README.rst

here is the link to the example:
https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html?highlight=doc_source_url